lib/lightning_web/live/components/icon.ex

```diff
@@ -14,6 +14,119 @@ defmodule LightningWeb.Components.Icon do
     """
   end
 
+  def project(assigns) do
```

Comment:
Is there a reason you're not using the `.outer_svg` helper with these new functions?

Reply:
Nope, no specific reason. Changed it.